### ISSUE TITLE
Inaccurate Network Graph Data Leading to Payment Failures in Zeus Wallet v0.9.0

### ISSUE THREAD

**Issue by Perlover** [ZeusLN/zeus#2402]
**Issue Description**:

There seems to be a bug in the Zeus wallet related to the network graph data. I've noticed that many of my recent payments have failed, and upon investigating, I discovered the following:

- When viewing the network graph data in the wallet, it shows approximately 40,000 nodes and 132,000 channels.
- However, in reality, the number of nodes and channels is about three times lower.
  
To further investigate, I switched to the EGS provider in the settings and selected Blixt as the source. After restarting the wallet, it then displayed more realistic values of around 18,000 nodes and 40,000 channels.

If I switch back to the Zeus source and restart the app, it again shows 40,000 nodes and 132,000 channels. This suggests that your server, which provides data for Express Graph Sync, contains an excessive amount of inaccurate or redundant data. This likely leads to the high failure rate in payments, as too many routes are not being found.

**Steps to reproduce**:
1. Use Zeus wallet v0.9.0.
2. Check the network graph data (should show ~40,000 nodes and ~132,000 channels).
3. Switch to EGS provider and select Blixt as the source.
4. Restart the wallet and check the network graph data (should show ~18,000 nodes and ~40,000 channels).
5. Switch back to Zeus source and restart the app.
6. The inaccurate, inflated data will reappear.

This issue might be the reason why many payments are not completing, as invalid or excessive routing data is being used.

**Comment by kaloudis** [comment]
Thanks for reporting. The speedloader server has been restarted. We'll look to automate the process moving forward.

**Comment by Perlover** [comment]
Now it works! Thanks!

**Comment by Perlover** [comment]
Now there is the same problem....

37K nodes, 105K channels...

**Comment by kaloudis** [comment]
Making great progress on our new Graph Sync server. Hoping to pop it in as a replacement some time next week after testing.

In the meantime, we'll be keeping an eye on the current server an resetting it when appropriate.

**Comment by kaloudis** [comment]
New express graph sync serer is finally in place!

**Comment by Perlover** [comment]
Version of the Zeus wallet: 0.9.1.

I believe the issue has reappeared, as I am currently unable to process payments in places where I would normally pay without any problems. Most errors show either "incorrect payment details" or "route not found."

Afterward, I checked the network information. Right now, it shows 22,000 nodes and 48,000 channels. However, just recently, as of yesterday, I observed 18,000 nodes and around 33,000 channels. So, I suspect that your database of nodes and channels has become desynchronized again. As a result, wallets relying on accelerated synchronization with your database are once again unable to function properly.

**Comment by Perlover** [comment]
I also tried running a few experiments.

First, I checked the logs when my payments were failing. In my recent experiments, there was an error from the onion package, showing "expire too soon." As far as I understood, some nodes were likely out of sync. It seemed like a synchronization issue, possibly with the wallet having outdated data from the Bitcoin network. L

Then, I switched the graph source to the Blixt server, and the payment went through successfully. After that, I switched back to the Zeus server. In all cases, I reset the EGS database.

Afterward, payments started going through on your network graph as well. Restarting the application may have helped to some extent.

Thanks for your attention!
